db/seeds.rb

```diff
@@ -1,10 +1,310 @@
+  description: "Deluxe edition, signed by the artists!! A unique piece of art.",
+  condition: "Like new",
+  owner: christophe
+)
+
+
+
+file  = File.open(Rails.root.join('db/media/vinyls/alive.jpg'))
+
+christophe_vinyl_5.photo.attach(io: file, filename: 'alive.jpg', content_type: 'image/jpeg')
+
+christophe_vinyl_5.save!
+
+
+bastien_vinyl_1 = Vinyl.new(
+  title: "Random access memories",
+  artist: "Daft Punk",
+  price_per_day: 6,
+  description: "A state of the art vinyl of the most famouse Daft Punk album, vintage.",
+  condition: "Good",
+  owner: bastien
+)
+
+file  = File.open(Rails.root.join('db/media/vinyls/memories.jpg'))
+
+bastien_vinyl_1.photo.attach(io: file, filename: 'memories.jpg', content_type: 'image/jpeg')
+
+bastien_vinyl_1.save!
+
+
+
+bastien_vinyl_2 = Vinyl.new(
+  title: "Discovery",
+  artist: "Daft Punk",
+  price_per_day: 4,
+  description: "My favorite album from Daft Punk, but unfortunately a bit used.",
+  condition: "Acceptable",
+  owner: bastien
+)
+
+file  = File.open(Rails.root.join('db/media/vinyls/discovery.jpg'))
+
+bastien_vinyl_2.photo.attach(io: file, filename: 'discovery.jpg', content_type: 'image/jpeg')
+
+bastien_vinyl_2.save!
+
+
+
+corentin_vinyl_1 = Vinyl.new(
+  title: "Untouchables",
+  artist: "Korn",
+  price_per_day: 8,
+  description: "Untouchables debuted at number two on the Billboard 200 with 434,000 copies sold during its first week.",
+  condition: "Very good",
+  owner: corentin
+)
+
+
+file  = File.open(Rails.root.join('db/media/vinyls/korn.jpg'))
+
+bastien_vinyl_2.photo.attach(io: file, filename: 'korn.jpg', content_type: 'image/jpeg')
+
+bastien_vinyl_2.save!
+
+puts "Seeding some rentings..."
+
+total_price = 7 * christophe_vinyl_1.price_per_day
+
+renting_1 = Renting.create!(
+  renter: corentin,
+  vinyl: christophe_vinyl_1,
+  start_date: Date.today,
+  end_date: Date.today + 7,
+  status: "pending",
+  total_price: total_price
+)
+
+total_price = 6 * christophe_vinyl_2.price_per_day
+
+renting_2 = Renting.create!(
+  renter: corentin,
+  vinyl: christophe_vinyl_2,
+  start_date: Date.today,
+  end_date: Date.today + 6,
+  status: "pending",
+  total_price: total_price
+)
+
+total_price = 10 * christophe_vinyl_5.price_per_day
+
+renting_3 = Renting.create!(
+  renter: bastien,
+  vinyl: christophe_vinyl_5,
+  start_date: Date.today,
+  end_date: Date.today + 10,
+  status: "pending",
+  total_price: total_price
+)
+
+total_price = 5 * bastien_vinyl_1.price_per_day
+
+renting_4 = Renting.create!(
+  renter: lise,
+  vinyl: bastien_vinyl_1,
+  start_date: Date.today,
+  end_date: Date.today + 5,
+  status: "accepted",
+  total_price: total_price
+)
+
+total_price = 1 * corentin_vinyl_1.price_per_day
+
+renting_5 = Renting.create!(
+  renter: christophe,
+  vinyl: corentin_vinyl_1,
+  start_date: Date.today,
+  end_date: Date.today + 1,
+  status: "declined",
+  total_price: total_price
+)
+
+puts "Finished!"
+
+
+
+
+# total_price: (end_date - start_date) * price_per_day
```

Comment:
missing empty line at the end of the file

Reply:
Noted